packages/augur-ui/src/modules/modal/containers/modal-create-market.ts

```diff
@@ -18,7 +18,7 @@ const mapStateToProps = (state: AppState) => ({
 const mapDispatchToProps = (dispatch: ThunkDispatch<void, any, Action>) => ({
   closeModal: () => dispatch(closeModal()),
   submitNewMarket: (data: NewMarket, cb: NodeStyleCallback) =>
-    dispatch(submitNewMarket(data, cb)),
+    dispatch(submitNewMarket(data, null, false, cb)),
```

Comment:
example of a the place to have more descriptive method instead of passing in null, false
just create a new pass through method `createNewMarket` as an example